templates/secure-connections.html

```diff
@@ -0,0 +1,71 @@
+<p>The following visualization shows what information is visible to eavesdroppers with and without Tor Browser and HTTPS encryption:</p>
+
+<ul>
+  <li>Click the “Tor” button to see what data is visible to observers when you're using Tor. The button will turn green to indicate that Tor is on.</li>
+  <li>Click the “HTTPS” button to see what data is visible to observers when you're using HTTPS. The button will turn green to indicate that HTTPS is on.</li>
```

Comment:
If you add new strings this way, they will not be translatable through the lektor-i18n plugin. Please instead of:

`Click the “HTTPS” button to see what data is visible to observers when you're using HTTPS. The button will turn green to indicate that HTTPS is on.` 

do:

`{{ _("Click the 'HTTPS' button to see what data is visible to observers when you're using HTTPS. The button will turn green to indicate that HTTPS is on.") }} `

Reply:
So that's what that syntax means. That's changed now.